apps/web/frontend/components/topNavigation/topNavigation.tsx

```diff
@@ -33,14 +33,12 @@ export const TopNavigation = (): JSX.Element => {
         {session.status !== 'authenticated' && (
           <TopNavigationLink onClick={() => signIn('github')}>Sign in</TopNavigationLink>
         )}
-
         <TopNavigationLink href="/time">Time</TopNavigationLink>
         <TopNavigationLink href="/sheet">Sheet</TopNavigationLink>
+        <div className="h-[20px]  w-[1px] bg-blue-400" />
```

Comment:
The pipe looks a little bit weird on light mode:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/1886867/224671913-ffc85a94-626d-4d19-88e4-229b6be4e03d.png">

Please make it gray on light mode.